com.github.rhyme_player.rhyme.yml

```diff
@@ -0,0 +1,67 @@
+app-id: com.github.rhyme_player.rhyme
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
```

Comment:
Why did you change it back? `runtime-version` and `base-version` should both be `21.08`.